modules/isofs/imports.lst

```diff
@@ -9,6 +9,7 @@ ioman_IMPORTS_start
 I_open
 I_close
 I_read
+I_write
```

Comment:
Writing to iso files?

Reply:
oh this is a leftover from when I was writing log files, can be removed.